phoenix.pro

```diff
@@ -213,8 +213,13 @@ include(pri/program.pri)
 include(pri/qtsysteminfo.pri)
 
 contains(DEFINES, QUAZIP_INSTALLED) {
-    INCLUDEPATH += /usr/include/quazip
-    LIBS += -lquazip
+    greaterThan(QT_MAJOR_VERSION, 4):exists(/usr/include/quazip5) {
+        INCLUDEPATH += /usr/include/quazip5
```

Comment:
Qt4 is unsupported since 2015... we can not support it anymore, in fact we have to hurry so Fritzing can be ready for Qt6 by next year. So no such switches needed.

Reply:
Removed Qt 4 code.